### ISSUE TITLE
Unsafe use of json_extract() error when querying on a view

### ISSUE THREAD

**Issue by mbornstein** [Airsequel/Airsequel-Issues#2]
I noticed the following regression 1 or 2 days ago:

I have the following view:

```sql
CREATE VIEW "cleaned_json_store" AS select distinct
  ...
  json_extract(json, "$.lastUpdatedTime") as updated
from json_store
order by updated
```

When I want to add other views (or any queries really) building on top of the view above, I get the error

```sql
select * from cleaned_json_store
```

`Parse error near line 6: unsafe use of json_extract()`

This used to work before but now sure how I can find or report the build nr. or version of airsequel I am using. I wasn't able to find this specific error message online, wondering if this is an airsequel error or sqlite error. It might also help to get a bit more information about the error or a way to inspect what the final query was to which the error line etc. relates to.

**Comment by ad-si** [comment]
Thanks for the report! This was already fixed in the main branch and will be released with v0.5 😊

**Comment by ad-si** [comment]
I just realized that it's actually a feature! 😅

We restricted what kind of SQL queries users are allowed to run -> https://www.sqlite.org/pragma.html#pragma_trusted_schema
And apparently `json_extract` isn't safe …

We will add a setting to disable this pragma on a per database basis to a future version of Airsequel Enterprise. I'll keep you posted.

**Comment by ad-si** [comment]
Corresponding ticket => https://github.com/Airsequel/Airsequel-Issues/issues/3

**Comment by ad-si** [comment]
That `json_extract` isn't considered safe is actually a bug and was fixed today in SQLite => https://sqlite.org/forum/forumpost/762412dcf0cf07b4

**Comment by ad-si** [comment]
This was fixed in v0.5, which includes SQLite v3.41 (https://blog.airsequel.com/airsequel-0-5-improved-spreadsheet-ui-ai-powered-sql-query-generation)
